src/main/kotlin/it/pagopa/wallet/audit/LoggingEvent.kt

```diff
@@ -18,17 +18,15 @@ data class WalletDeletedEvent(val walletId: String) : LoggingEvent()
 
 data class SessionWalletCreatedEvent(val walletId: String) : LoggingEvent()
 
-data class WalletPatchEvent(val walletId: String) : LoggingEvent()
+data class WalletPatchEvent(
+    val walletId: String,
+    val updatedApplications: List<AuditWalletApplication>
+) : LoggingEvent()
 
 data class WalletDetailsAddedEvent(val walletId: String) : LoggingEvent()
 
-data class WalletNotificationEvent(
-    val walletId: String,
-    val validationOperationId: String,
-    val validationOperationResult: String,
-    val validationOperationTimestamp: String,
-    val validationErrorCode: String?,
-) : LoggingEvent()
+data class WalletNotificationEvent(val walletId: String, val auditWallet: AuditWallet) :
```

Comment:
For the same reason as above, it can be `WalletOnboardCompleted` (success or failure depends on operationResults, but the onboarding flow is completed anyway)

Reply:
Applied in e4d2913f